src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDK.scala

```diff
@@ -118,7 +122,100 @@ class TextAnalyticsLanguageDetection(override val textAnalyticsOptions: Option[T
     }
 }
 
+object TextSentimentV4 extends ComplexParamsReadable[TextSentimentV4]
+class TextSentimentV4(override val textAnalyticsOptions: Option[TextAnalyticsRequestOptions] = None,
+                      override val uid: String = randomUID("TextSentimentV4"))
+  extends TextAnalyticsSDKBase[SentimentScoredDocumentV4](textAnalyticsOptions)
+    with HasConfidenceScoreCol {
+  logClass()
+
+  override def outputSchema: StructType = SentimentResponseV4.schema
+
+  override protected val invokeTextAnalytics: String => TAResponseV4[SentimentScoredDocumentV4]= (text: String) =>{
+    val textSentimentResultCollection = textAnalyticsClient.analyzeSentimentBatch(
+      Seq(text).asJava, "en", textAnalyticsOptions.orNull)

```

Comment:
Shouldn't "en" be a param rather than hardcoded here?

Reply:
This is just the first happy path PR. More changes coming shortly in future PRs. Thank you!